### PR TITLE
[terra-status-view] padding fix for orion-fusion-theme to reduce white space

### PR DESCRIPTION
**Pull request by miyagiborn**
### Summary

**What was changed:**
Changed the padding values around the status view for orion-fusion-theme

**Why it was changed:**
To reduce the unnecessary white space, based on client feedback.

### Testing

This change was tested using:

- [x] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [x] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

CHARTING-35410
---

### Screenshots
**Before:**
Chrome:
Chrome_before__status_view_padding_orion_fusion_theme

Edge:
image

IE11 (issue does not seem to be occurring in IE11, theme values do not appear to be honoured. Looks to be a separate issue)
image

**After:**
Chrome
Chrome_after_status_view_padding_orion_fusion_theme

Edge
image

Jest:
image

Lint:
image

---
Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/CHANGELOG.md
Comment (supreethmr) [inline packages/terra-core-docs/CHANGELOG.md:6]:
```suggestion
```

why do we need change log update on core-docs ..??

File: packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss
Comment (supreethmr) [inline packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss:6]: are these values are updated from UX ??

Comment (leducquan) [reply packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss:6]: We will require UX review, these were the suggested values as per
https://teams.microsoft.com/l/message/19:3ed4d4d6ffba48138550b9a91a2b50b4@thread.skype/1683727361903?tenantId=fbc493a8-0d24-4454-a815-f4ca58e8c09d&groupId=45f47bfa-38a9-444d-8933-9c20266ae38c&parentMessageId=1683727361903&teamName=Terra&channelName=UX%20Support&createdTime=1683727361903&allowXTenantAccess=false

File: packages/terra-core-docs/CHANGELOG.md
Comment (leducquan) [reply packages/terra-core-docs/CHANGELOG.md]: removed

File: packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss
Comment (leducquan) [reply packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss:6]: We are receiving negative feedback about all the unnecessary whitespace in the orion fusion theme in Charting Navigator. Please see screenshots above. It was pointed out by @jeremyfuksa that the original values were potentially a bug.

**Comment by leducquan** [comment]
> Can you share Jira details for terra-status-view (number and group name. not link). Because there is another PR which is related to updates of orion-fusion-theme for terra-status-view. so just thinking this could be an duplicate of #3799

@supreethmr  We don't have a jira as we thought small contributions didn't require one. Do we need one?

**Comment by gerard-isdell** [comment]
Hi @supreethmr. I don't think this is a duplicate. This PR updates the padding, whereas https://github.com/cerner/terra-core/pull/3799 updates the font-size.

**Comment by sdadn** [comment]
> > Can you share Jira details for terra-status-view (number and group name. not link). Because there is another PR which is related to updates of orion-fusion-theme for terra-status-view. so just thinking this could be an duplicate of #3799
> 
> @supreethmr We don't have a jira as we thought small contributions didn't require one. Do we need one?

@leducquan A Jira would be preferable for tracking purposes around the change. We also try to avoid internal links on open source PRs so you can link the Teams conversation there instead.

File: packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss
Comment (sdadn) [reply packages/terra-status-view/src/orion-fusion-theme/StatusView.module.scss:6]: Looks like these values are from Jeremy who had past UX contributions to Terra, so the change looks reasonable and fine. I wouldn't be too worried about breaking WDIO/Jest snapshots as that's not part of our definition of breaking change (otherwise we won't be able to make any changes at all!!). This PR would still need an approval from a UX expert before it can be merged on top of engineering approvals.

**Comment by leducquan** [comment]
Added Chrome, Edge and IE11 screenshots. Do not have access to Safari.

IE11 appears to not display the padding issue - seems it is not honouring the orion-fusion-theme values. Looks like it might be a separate issue.

**Comment by sycombs** [comment]
Because you are opening this PR from a fork, it cannot build on our CI system. Can you run the Jest tests and linter locally and attach a screenshot showing that they are passing? `npm run jest` and `npm run lint`.

**Comment by leducquan** [comment]
> Because you are opening this PR from a fork, it cannot build on our CI system. Can you run the Jest tests and linter locally and attach a screenshot showing that they are passing? You can use `npm run jest` and `npm run lint`.

I have attached lint and jest screenshots.

**Review by sycombs — APPROVED** [review]

**Review by sdadn — APPROVED** [review]
Looks good from an engineering standpoint. But this will need UX review before it can be merged.

**Comment by leducquan** [comment]
Is there anyone we need to pull in from UX to get this one moving? @sdadn @sycombs  Thanks

**Comment by sdadn** [comment]
> Is there anyone we need to pull in from UX to get this one moving? @sdadn @sycombs Thanks
@leducquan An approval from @jeremyfuksa would be great!

**Review by jeremyfuksa — APPROVED** [review]

**Comment by leducquan** [comment]
@sdadn @supreethmr @sycombs Just checking if this has been released/ tagged for release? Thanks

**Comment by sdadn** [comment]
> @sdadn @supreethmr @sycombs Just checking if this has been released/ tagged for release? Thanks

Hmm that's odd, the changelog entries from this pr aren't in main which is probably why a release didn't get triggered. I'll look at what happened there.

**Comment by sdadn** [comment]
Okay, looks like the changes are in main but haven't been released. 
Release PR: https://github.com/cerner/terra-core/pull/3843

**Comment by leducquan** [comment]
> Okay, looks like the changes are in main but haven't been released. Release PR: #3843

@sdadn , I don't see terra-status-view in Release PR: #3843

**Comment by sdadn** [comment]
> @sdadn , I don't see terra-status-view in Release PR: #3843

Yeah for some reason, terra-status-view is getting ignored. I'm currently investigating it.

**Comment by sdadn** [comment]
@leducquan Had to do a manual release but finally released [terra-status-view@4.56.0](https://www.npmjs.com/package/terra-status-view)! Let me know if this works and has the changes you need.

**Comment by leducquan** [comment]
> @leducquan Had to do a manual release but finally released [terra-status-view@4.56.0](https://www.npmjs.com/package/terra-status-view)! Let me know if this works and has the changes you need.

Thanks for your assistance @sdadn , it worked.

**Pull request merged**
